client/app/lib/statefulForm.js

```diff
@@ -0,0 +1,127 @@
+/** @jsx h */
+import { h, Component } from 'preact'
```

Comment:
I just think if there is some JSX, we can use `.jsx`. And we could have helper/library in `.jsx`?